.template/Gemfile.lock

```diff
@@ -2,64 +2,70 @@ GEM
   remote: https://rubygems.org/
   specs:
     ast (2.4.2)
-    diff-lcs (1.4.4)
+    diff-lcs (1.5.0)
     docker-api (2.2.0)
       excon (>= 0.47.0)
       multi_json
-    excon (0.90.0)
+    excon (0.100.0)
```

Comment:
Need to upgrade the excon to fix the issue on https://github.com/nimblehq/rails-templates/actions/runs/5441234668/jobs/9894962870

```
An error occurred in a `before(:suite)` hook.
Failure/Error: Docker::Container.get(container_id.strip)
ArgumentError:
  The `:host` parameter should not be set for `unix://` connections.
  When supplying a `unix://` URI, it should start with `unix:/` or `unix:///`.
```